package.json

```diff
@@ -39,6 +39,14 @@
     "dependencies": {
         "whatwg-url": "^5.0.0"
     },
+    "peerDependencies": { 
+        "encoding": "*"
```

Comment:
I think it's better to be explicit here? Since the package is still in `0.x` phase it seems very possible that there will be breaking changes ahead...

```suggestion
        "encoding": "^0.1.0"
```

Reply:
Yes, I'll change that.